api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/v3/EligibilityAndEntitlementServiceV3Test.java

```diff
@@ -68,35 +65,18 @@
     EligibilityAndEntitlementDecisionFactory eligibilityAndEntitlementDecisionFactory;
 
     @Test
-    void shouldReturnExistingClaimIdWhenLiveClaimAlreadyExists() {
+    void shouldReturnDuplicateWhenLiveClaimAlreadyExists() {
         //Given
         UUID existingClaimId = UUID.randomUUID();
-        setupCommonMocks(existingClaimId);
-        EligibilityAndEntitlementDecision decisionResponse = setupEligibilityAndEntitlementDecisionFactory(INELIGIBLE);
+        given(claimRepository.findLiveClaimWithNino(any())).willReturn(Optional.of(existingClaimId));
 
         //When
         EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementServiceV3.evaluateNewClaimant(CLAIMANT);
 
         //Then
-        assertThat(decision).isEqualTo(decisionResponse);
-        verifyCommonMocks(existingClaimId, NOT_DUPLICATE);
-        verifyNoInteractions(duplicateClaimChecker);
-    }
-
-    @Test
-    void shouldReturnEligibleWhenNotDuplicateAndEligible() {
```

Comment:
I don't understand why this test has been removed, I think this stems back to why the changes have been made to this service, it feels like if you wanted to change this it should have been in its own PR, it doesn't seem to have anything to do with remove update functionality.

Reply:
It hasn't been removed, the diff on github is showing it that way. If you scroll down you can see the test is still there.